### PR TITLE
1.2.5 engine release

### PR DESCRIPTION
**Pull request by elliotsinclair**
Small update to website as customers are now receiving transform.engines on version 1.2.5.

### CODE REVIEW

**Review by henryharrod — COMMENTED** [review]
I wonder if the best way to do this might be to keep the 'current' engine version as 1.2.1 )(as that's what's availble to download) and add an 'info' or 'note' explaining that some units out of the factory might show a version 1.2.5 - with an explanitory sentance or two?

**Comment by elliotsinclair** [comment]
Agree! Just added some edits, think that works better?

**Review by henryharrod — COMMENTED** [review]
I think we can probably just link 1.2.5 to 1.2.1 page and drop the new .md file?

**Comment by elliotsinclair** [comment]
Done!

File: info/downloads/index.md [hunk 1]
```diff
@@ -35,14 +35,21 @@ compatibility ranges at any time.</small>
...
 | 1.2.0          | 1.2.0                     | 30/08/2024   |         | [Release notes](v1-2-0.md) |
 |                | 1.1.1                     | 04/07/2024   |         | [Release notes](v1-1-1.md) |
 | 1.1.0          | 1.1.0                     | 24/06/2024   |         | [Release notes](v1-1-0.md) |
 |                | 1.0.1                     | 22/04/2024   |         | [Release notes](v1-0-1.md) |
 | 1.0.0          | 1.0.0                     | 22/04/2024   |         | [Release notes](v1-0-0.md) |
 
-:::note
+:::note 
+
+The latest **transform**.engine's will ship from the factory with Engine Version, 1.2.5. these are purely manufacturing changes, and Engine Version 1.2.1 is still the latest version to download.
```
Comment (henryharrod) [inline info/downloads/index.md:48]:
```suggestion
The latest **transform**.engines will ship from the factory with Engine Version, 1.2.5. these are purely manufacturing changes, and Engine Version 1.2.1 is still the latest version to download.
```

Comment (henryharrod) [reply info/downloads/index.md:48]: Nitpick

**Review by henryharrod — APPROVED** [review]
Just one small nitpick - then good to go! Thanks!

File: info/intro/roadmap.md [hunk 2]
```diff
@@ -11,9 +11,15 @@ feedback we receive from our ~~adoring fans~~ users.
 
 | Feature Ref. | Feature Details                                                                                                                           | Planned For   |
 | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| TF-157       | Chain Groups - ways to slice and dice your chains. | End of 2024 |
-| FA-598       | Initial console integration - control your plugins from a console! | End of 2024 |
-| TF-1659      | Network Redundancy - mirror two **transform**.engines so that they can be used in a redundant configuration.                              | 2025          |
+| FA-598       | Initial console integration - control your plugins from a console! | Q1 2025 |
```
Comment (henryharrod) [inline info/intro/roadmap.md:14]: As per chat message

**Review by henryharrod — CHANGES_REQUESTED** [review]

**Review by henryharrod — APPROVED** [review]

**Pull request merged**